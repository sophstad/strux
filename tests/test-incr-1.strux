int main() {
  num i = 0.0;
  int j = 1;
  print(i++);
  print(j++);
  return 0;
}
