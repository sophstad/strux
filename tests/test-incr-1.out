1.000000
2
